### PR TITLE
Build "dotnot"

### PR DESCRIPTION
**Pull request by Duffleman**

### CODE REVIEW

File: lib/dotnot/dotnot_test.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	},
+	{
+		name: "Dotnotate a bunch of levels",
+		from: `{"nest":{"two":{"three":{"four":"layers_deep"}},"two_half":{"and_siblings_too":true}},"root":"key"}`,
+		to:   `{"nest.two.three.four":"layers_deep","nest.two_half.and_siblings_too":true,"root":"key"}`,
+	},
+	{
+		name: "Dotnotate arrays safely",
+		from: `{"nest":{"two":["arrays","must","make","it","through"],"two_half":{"and_siblings_too":true}},"root":"key"}`,
+		to:   `{"nest.two":["arrays","must","make","it","through"],"nest.two_half.and_siblings_too":true,"root":"key"}`,
```
Comment (billinghamj) [inline lib/dotnot/dotnot_test.go:36]: What will it do with arrays containing objects?

Comment (Duffleman) [reply lib/dotnot/dotnot_test.go:36]: You're testing my skills right? Don't think I won't rise to the challenge

Comment (Duffleman) [reply lib/dotnot/dotnot_test.go:36]: https://github.com/cuvva/cuvva-public-go/pull/18/commits/109363f752f0b77b317527338e50ad87ae8b3901

Comment (billinghamj) [reply lib/dotnot/dotnot_test.go:36]: oof, very nice 👌

**Comment by billinghamj** [comment]
@Duffleman Could you please finish this one up or close?

**Comment by Duffleman** [comment]
Closing because @mfiedorowicz found a better/smarter third party lib

**Comment by billinghamj** [comment]
@Duffleman care to share/link? :P

**Comment by Duffleman** [comment]
https://github.com/jeremywohl/flatten

**Pull request closed without merging**